### DISCUSSION TITLE
legends out of content in Safari

### DISCUSSION THREAD

**Discussion by SueSamuel** [apexcharts/apexcharts.js#2778] [Q&A]
Hi, I have this situation that when I see these graphics in Safari, the legends are outside the div, but when I render it in chrome, this does not happen, I would not know what it could be.
![Screen Shot 2021-11-12 at 9 01 59 AM](https://user-images.githubusercontent.com/31971419/141479120-81400024-dc64-4398-a891-2db5e8ae8a26.png)
**THIS IS SAFARI**
![Screen Shot 2021-11-12 at 9 02 55 AM](https://user-images.githubusercontent.com/31971419/141479252-1ea121a1-325d-4161-9807-7ab50cdd95fc.png)
**THIS IS IN CHROME.**

**Comment by patrickaroo** [comment]
Having this issue as well.
